src/api/desktop/recent-saves/inputs.ts

```diff
@@ -40,8 +40,14 @@ export const setDefaultsAndCoerceTypes = (
     ...stringParams,
   };
 
+  let count = parseInt(withDefaults.count, 10);
+  if (count > 20) {
+    // versions of Firefox are asking for 88 items, so we'll allow up to 20
+    count = 20;
```

Comment:
could you add a comment as to why we need to limit to 20?

Reply:
@jpetto I don't know why. This is what the code was doing accepting before and was before me.

Reply:
🤷 okay!